core/commands/IssueFileAddCommand.php

```diff
@@ -0,0 +1,185 @@
+<?php
+# MantisBT - A PHP based bugtracking system
+
+# MantisBT is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MantisBT is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MantisBT.  If not, see <http://www.gnu.org/licenses/>.
+
+require_api( 'authentication_api.php' );
+require_api( 'bug_api.php' );
+require_api( 'constant_inc.php' );
+require_api( 'config_api.php' );
+require_api( 'helper_api.php' );
+require_api( 'user_api.php' );
+
+use Mantis\Exceptions\ClientException;
+
+/**
+ * Command data
+ * - In files, `error` and `size` are used only by web UI.
+ * - Reporter is optional, it is defaulted to logged in user.
+ *
+ * Sample:
+{
+  "query": {
+    "issue_id": 12345
+  },
+  "payload": {
+    "reporter": {
+      "name": "vboctor"
+    },
+    "files": [
+      {
+        "name": "filename.ext",
+        "type": "application/...",
+        "tmp_name": "/tmp/php/phpRELws8",
+        "error": 0,
+        "size": 114
+      }
+    ]
+  }
+}
```

Comment:
This is not valid PHPDoc format, lines should begin with ` * `

Reply:
Fixed.